server/src/app.ts

```diff
@@ -14,10 +14,24 @@ import {SubscriptionServer} from "subscriptions-transport-ws";
 import {createServer} from "http";
 import {findUserByID} from "./database";
 import flash = require("connect-flash");
+const fetch = require('isomorphic-fetch')
+const bodyParser = require('body-parser')
 
 // Set up Express and its middleware
 export let app = express();
 
+app.post("/api/slack/feedback", bodyParser.json(), (req, res) => {
+  fetch("https://hooks.slack.com/services/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", {
+    method: "POST",
+    body: JSON.stringify(req.body)
```

Comment:
You probably shouldn't send the body containing just anything to Slack (insert elements of the client's request data into pre-determined slots in a template).

It'd also be nice if the webhook message contained fields for the user's UUID, name, email, and Slack username (all of which are available on the request since only signed in users should be able to access this endpoint) as well as the URL of the page they were viewing.  Slack docs for formatting webhook messages are at https://api.slack.com/docs/message-formatting